dtests/src/resources/scripts/simulateFileStream

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env sh
+#set -vx
+
+# This script helps to simulate twitter stream as a file stream. It copies 
+# tweet files from one folder to another folder on which a file stream can be
+# created for about 10 mins.
+
+if [ $# -eq 0 ]; then
+    echo "ERROR: incorrect argument specified: " "$@"
+    echo "Usage:./simulateFileStream.sh <DESTINATION_FOLDER> <SNAPPYDATA_BUILD_PATH>"
+    exit 1
+fi
+
+# Removing directories from user specified location if exists and creating new ones.
+rm -rf "$1"/twitterdata
+rm -rf "$1"/copiedtwitterdata
+mkdir -p "$1"/twitterdata
+mkdir -p "$1"/copiedtwitterdata
+
+# Extract preloaded tweets
+tar -zxf "$2"/quickstart/data/tweetdata.tgz -C "$1"/twitterdata
+FILES="$1"/twitterdata/tweets_*/tweets-*
+
+# Copy tweets to a folder to simulate a file stream.
+for file in $FILES
```

Comment:
FileDStream and twitterStream use cases can be simulated completely in Hydra without the script requirement. user can pass couple of parameters and can write a test without worrying about this  implementation or by  just providing the code that needs to be executed on stream . for now this is okay and this can be handled  later